packages/core/stories/overlay/overlay.qa.stories.tsx

```diff
@@ -152,9 +168,19 @@ export const NoStyleInjectionCloseButton: StoryFn<
           <Button>Show Overlay</Button>
         </OverlayTrigger>
         <OverlayPanel>
-          <OverlayPanelCloseButton />
```

Comment:
Similar to dialog, need to keep at least 1 `OverlayPanelCloseButton` for backwards compatibility check